ui/src/app/utils/index.tsx

```diff
@@ -531,12 +531,12 @@ export function highlightSearchTerm(searchTerm: string, stringToHighlight: strin
 
 // render a float value as US currency, rounded to cents: 255.372793 -> $255.37
 // negative values are rendered as $0
-export function renderUSD(value: number) {
+export function formatFreeCreditsUSD(value: number): string {
   value = value || 0.0;
   if (value < 0.0) {
-    return <div style={{fontWeight: 600}}>$0</div>;
+    return '$0';
```

Comment:
nit: if the value is exactly 0, it will print as `$0.00` but if negative it's `$0`?

Reply:
switching to `<=` makes sense